frontend/components/pages/BuildFormPage/step-1.component.tsx

```diff
@@ -55,6 +58,10 @@ interface IStep1Props {
   goToNextStep: () => void
 }
 
+const stripNonAlpha = (str: string): string => str.replace(/[^0-9a-z\-_]/gi, "")
+const stripIcons = (str: string): string =>
+  str.replace(/[[A-z-]+=[A-z-]+\]/gi, "")
```

Comment:
Not sure if this is only wip and you're on it, but stripping non alpha is probably not a good idea. You'll want to transliterate instead. This package here looks like it would do a good job: https://www.npmjs.com/package/transliteration

Stripping the icons however is good, I actually missed that in the server side implementation!

Reply:
I've thought about it a little when implementing, this is when pre-filling the form, so I thought of leaving the field empty if it's in another alphabet/language no too bad. However I agree transliteration could be nice, and the package is very small.

I just hope that it won't translate unknown english/game terms to something weird.

Reply:
So, I ended up trying it, and the exported code is huge, imo it ends up a small edge-case and I don't want to increase bundle size that much just for this, we can revisit this later:

![image](https://user-images.githubusercontent.com/3461986/105618874-579ae000-5dba-11eb-9ce8-c1145349a8b8.png)